docs/basics.md

```diff
@@ -797,7 +797,11 @@ Also, you can use `within` inside a session, but you can't call session from ins
 Like in Mocha you can use `x` and `only` to skip tests or to run a single test.
 
 * `xScenario` - skips current test
+* `xScenario.skip` - skips current test
 * `Scenario.only` - executes only the current test
+* `xFeature` - skips current suite <Badge text="Since 2.7" type="warning"/>
```

Comment:
wasnt sure what to do with version warning

Reply:
we probably won't release 2.7 just for that, so this will go as 2.6.6